people/chooser.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from django.forms.models import modelform_factory
+from django.utils.translation import gettext_lazy as _
+
+from dal import autocomplete
+
+from generic_chooser.views import ModelChooserCreateTabMixin, ModelChooserMixin, ModelChooserViewSet
+from generic_chooser.widgets import AdminChooser
+from people.models import Person
+
+
+class PersonChooserMixin(ModelChooserMixin):
+    def get_unfiltered_object_list(self):
```

Comment:
When using in Watch/Paths, must remember to retain `get_object_list` in order for authorization to filter the users properly.